template/src/main.js

```diff
@@ -591,20 +591,15 @@ function init () {
   $('#scrollingNav .sidenav-search input.search').focus();
 
   /**
-   * Filter search
+   * Filter search with a delay of 200 ms
```

Comment:
I would remove the time from the doc comment, so if the time is changed in the code this comment doesn't become wrong. It would be better to explain *why* there is a delay. So `Filter search with a delay to prevent issues with very large projects hogging the browser event loop during the search`.

Reply:
My suggestion:
`Filter search with a debounce to decrease events` or
`Filter search with a debounce to fires only after the user stops typing`

Reply:
I prefer my suggestion still.